### PR TITLE
Tic tac toe assignment

### PR DESCRIPTION
**Pull request by sinha24**
## Description
- Added Tic tac toe in assignments

### Issue
#8 

**Screenshots;--**
screenshot 12

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Jo-ravar) [inline .idea/workspace.xml]: Do we really need this , what its use  write a documentation here.

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    </task>
+    <servers />
+  </component>
+  <component name="TimeTrackingManager">
+    <option name="totallyTimeSpent" value="1873000" />
+  </component>
+  <component name="ToolWindowManager">
+    <frame x="-9" y="-9" width="1938" height="1060" extended-state="6" />
+    <layout>
+      <window_info id="Project" active="true" anchor="left" auto_hide="false" internal_type="DOCKED" type="DOCKED" visible="true" show_stripe_button="true" weight="0.124801695" sideWeight="0.5" order="0" side_tool="false" content_ui="combo" />
```
Comment (Jo-ravar) [inline .idea/workspace.xml]: Add a a documentation here also , what is window_info  what is use ?

File: .idea/Tutorials.iml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Jo-ravar) [inline .idea/Tutorials.iml]: documentation needed.

File: assignments/F-01-93/.gitignore [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (Jo-ravar) [inline assignments/F-01-93/.gitignore]: we dont needed a git ignore  here . I have already added one

File: assignments/F-01-93/README.md [hunk 5]
```diff
@@ -0,0 +1,2164 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (Jo-ravar) [inline assignments/F-01-93/README.md]: We dont need this file

**Comment by Jo-ravar** [comment]
Add  screenshots for this PR.

**Comment by Jo-ravar** [comment]
use bullets to write what you have implemented and description of various files.

File: .idea/Tutorials.iml (hunk 3 above)
Comment (sinha24) [reply .idea/Tutorials.iml]: I have been using webstorm
.idea is its folder 
I'll add it to gitignore

File: assignments/F-01-93/.gitignore (hunk 4 above)
Comment (sinha24) [reply assignments/F-01-93/.gitignore]: Ok I'll add these to that gitignore and delete this

File: assignments/F-01-93/src/index.js [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+        [0, 1, 2],
+        [3, 4, 5],
+        [6, 7, 8],
+        [0, 3, 6],
+        [1, 4, 7],
+        [2, 5, 8],
+        [0, 4, 8],
+        [2, 4, 6],
+    ];
+    for (let i = 0; i < lines.length; i++) {
```
[the remark this answers is not in the document]
Comment (sinha24) [reply assignments/F-01-93/src/index.js]: Will use it from next time
Could not find any pattern here

**Comment by sinha24** [comment]
Requested changes Done.
Please see through.

File: assignments/F-01-93/README.md (hunk 5 above)
Comment (Jo-ravar) [reply assignments/F-01-93/README.md]: Remove this file.

**Comment by sinha24** [comment]
Required Changes done.
Please see through

**Pull request merged**